forge-gui/res/cardsfolder/upcoming/astarion_the_decadent.txt

```diff
@@ -0,0 +1,15 @@
+Name:Astarion, the Decadent
+ManaCost:4 W B
+Types:Legendary Creature Vampire Elf Rogue
+PT:4/4
+K:Deathtouch
+K:Lifelink
+T:Mode$ Phase | Phase$ End of Turn | ValidPlayer$ You | TriggerZones$ Battlefield | Execute$ TrigCharm | TriggerDescription$ At the beginning of your end step, ABILITY
+SVar:TrigCharm:DB$ Charm | Choices$ DBOppLoseLife,DBGainLife | CharmNum$ 1
+SVar:DBOppLoseLife:DB$ LoseLife | ValidTgts$ Opponent | LifeAmount$ X | TgtPrompt$ Choose target opponent to lose life equal to the amount of life they lost this turn | SpellDescription$ Target opponent loses life equal to the amount of life they lost this turn.
+SVar:DBGainLife:DB$ GainLife | Defined$ You | LifeAmount$ Y | SpellDescription$ You gain life equal to the amount of life you gained this turn.
+SVar:X:TargetedPlayer$LifeLostThisTurn
+SVar:Y:Count$LifeYouGainedThisTurn
+DeckHints:Type$Vampire|Elf|Rogue
```

Comment:
remove this line, Forge already gets this from `Type:` line above

Reply:
Removed redundant creature type deck hint line